applications/plugins/Geomagic/src/Geomagic/GeomagicDriver.cpp

```diff
@@ -334,10 +336,10 @@ void GeomagicDriver::initDevice()
     }
     
     // 2.6- Need to wait several ms for the scheduler to be well launched and retrieving correct device information before updating information on the SOFA side.
-    Sleep(42);        
+    std::this_thread::sleep_for(std::chrono::milliseconds(42));
```

Comment:
I can't test the behavior, but I assume the std lib method works and we don't need to be precise to the ms here.
I Just pushed a compilation fix on windows.